.pipelines/templates/template-run-rp.yml

```diff
@@ -0,0 +1,83 @@
+parameters:
+  workingDirectory: ""
+steps:
+  - script: |
+      cd ${{ parameters.workingDirectory }}
+
+      . secrets/env
+      echo "########## Building ARO RP Binary ##########"
+      go build ./cmd/aro
```

Comment:
I assume this is on a clean vm? and it doesn't matter if this process is running after the test has completed?

Reply:
indeed